platform/darwin/test/MGLExpressionTests.mm

```diff
@@ -712,6 +712,31 @@ - (void)testInterpolationExpressionObject {
     }
 }
 
+- (void)testInterpolationExpressionObjectWithEmptyStopsDictionary {
+    {
+        NSDictionary *stops = @{};
+        NSExpression *expression = [NSExpression expressionWithFormat:@"mgl_interpolate:withCurveType:parameters:stops:(x, 'cubic-bezier', { 0.42, 0, 0.58, 1 }, %@)", stops];
+        @try {
```

Comment:
It may be more clear using `XCTAssertThrowsSpecificNamed(expression.mgl_jsonExpressionObject, NSException, NSInvalidArgumentException)` instead.

Reply:
Thanks for introducing me to that one! Saves a ton.